charts/spire/templates/ncn/deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
           args:
             - '-x'
             - '-c'
-            - 'touch /token/{{ .Values.ncn.filename }} /token/spire-agent.conf /token/bundle.crt; chown 100:101 /token/{{ .Values.ncn.filename }} /token/spire-agent.conf /token/bundle.crt'
+            - 'touch /token/{{ .Values.ncn.filename }} /token/spire-agent.conf /token/bundle.crt; chown 100:101 /token /token/{{ .Values.ncn.filename }} /token/spire-agent.conf /token/bundle.crt'
```

Comment:
do you also need to touch /token directory here?

Reply:
I don't. I'm creating those files via touch if they don't exist.